instruqt/proxy/01-proxy-config/assignment.md

```diff
@@ -0,0 +1,61 @@
+---
+slug: proxy-config
+id: rkd0axhzc3me
+type: challenge
+title: Proxy Config
+teaser: Test connectivity and configure proxy
+notes:
+- type: text
+  contents: |
+    # Test external network access and configure proxy
+
+    This track environment has an isolated deployment host that can access external sites using a proxy server.
+tabs:
+- title: Host
+  type: terminal
+  hostname: isolated-host
+difficulty: basic
+timelimit: 1200
+---
+
+💡 Shell
+=========
+
+Test that the host has outbound traffic blocked.
+
+Test blocked website:
+```bash
+curl -v -m 5 https://lwn.net 2>&1 | egrep 'Trying|HTTP/|Failed|error|unreachable'
```

Comment:
How about just doing `curl https://www.google.com`

Reply:
I thought that lwn was cool :) I guess it makes no difference what url is used, any strong preference?